### PR TITLE
Load and clean data scripts

### PR DESCRIPTION
**Pull request by yixuangaoclara**
Please take a look at my scripts. Also, I am unsure whether I set the input/output path correctly in README.

### CODE REVIEW

File: scripts/clean_data.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        click.echo(df.info())
+        
+        # Remove duplicates
+        initial_count = len(df)
+        df = df.drop_duplicates()
+        final_count = len(df)
+        click.echo(f"Removed {initial_count - final_count} duplicate rows.")
+
+        # Save cleaned data
+        df.to_csv(output_path, index=False)
```
Comment (SimplyTim) [inline scripts/clean_data.py:23]: Maybe we should save the csv as a new file name?  Perhaps `<original_name_cleaned>.csv`?

File: scripts/clean_data.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import click
+import pandas as pd
+
+@click.command()
+@click.option('--input_path', type=str, help="Path to input CSV file to be cleaned.")
+@click.option('--output_path', type=str, help="Path where the cleaned file will be saved.")
+def clean_data(input_path, output_path):
```
Comment (SimplyTim) [inline scripts/clean_data.py:7]: I believe the standard practice is to keep this as `main()`, but I once the correct function is called below, and the script command works, I do not think it is a requirement.

Let me know what you think.

**Review by SimplyTim — COMMENTED** [review]
Just a few comments for consideration.

File: scripts/clean_data.py (hunk 1 above)
Comment (SimplyTim) [reply scripts/clean_data.py:23]: I see the file is being named in the passed argument. If this doesn't work, we will revisit later.

**Review by BryanLee06 — APPROVED** [review]
Looks good!

**Pull request merged**